receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -28,6 +28,10 @@ resource_attributes:
     description: The name of the index on a table.
     enabled: true
     type: string
+  postgresql.function.name:
+    description: The name of the function.
+    enabled: true
```

Comment:
it needs to be disabled by default when introduced.

Reply:
I replaced the resource attribute with metric attribute.